akka-actor-typed/src/main/scala/akka/actor/typed/internal/BehaviorImpl.scala

```diff
@@ -22,7 +22,7 @@ import scala.annotation.tailrec
   private def nullFun[T] = _nullFun.asInstanceOf[Any ⇒ T]
 
   implicit class ContextAs[T](val ctx: AC[T]) extends AnyVal {
-    def as[U] = ctx.asInstanceOf[AC[U]]
+    def as[U]: AC[U] = ctx.asInstanceOf[AC[U]]
```

Comment:
Can't it just be `ActorContext`? `AC` is bad, smell like `WC`.

Reply:
Please cross reference things otherwise things become unmanagable. 
You opened a ticket about this https://github.com/akka/akka/issues/24260

I don't disagree about AC perhaps being sub optimal, but isn't it all internal only? 
If its not, can we not simply avoid this alias?